scraping/scraping/middlewares.py

```diff
@@ -0,0 +1,52 @@
+from logging import getLogger
+from random import choice, random
+from time import monotonic
+from typing import Any, Optional
+
+from twisted.internet import reactor
+from twisted.internet.defer import Deferred
+
+from .utils import get_json_data
+
+logger = getLogger(__name__)
+user_agents = get_json_data("user_agents.json")
+
+AMAZON_MAX_REQUESTS_BEFORE_BREAK = 250
+AMAZON_MINIMUM_BREAK_TIME = 60**2 * 8  # 8 hours
+
+
+class AmazonSchedulerMiddleware(object):
+    """
+    stop scraping pages for  AMAZON_MINIMUM_BREAK_TIME to 2*AMAZON_MINIMUM_BREAK_TIME  seconds
```

Comment:
Maybe start the docstring with a capital first letter, and there are multiple double blank spaces within the docstring